sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/KnownLinuxVirtualMachineImage.java

```diff
@@ -21,20 +21,26 @@ public enum KnownLinuxVirtualMachineImage {
 
     /**
      * OpenSUSE-Leap 15.1.
-     * @deprecated use OPENSUSE_LEAP_15.
+     * @deprecated use OPENSUSE_LEAP_15_1.
      */
     @Deprecated
     OPENSUSE_LEAP_15_1("SUSE", "openSUSE-Leap-15-3", "gen1"),
-    /** OpenSUSE-Leap 15. */
+    /** OpenSUSE-Leap 15.
+     * @deprecated use OPENSUSE_LEAP_15.
+     */
+    @Deprecated
     OPENSUSE_LEAP_15("SUSE", "openSUSE-Leap-15-3", "gen1"),
```

Comment:
Please check if there's a minor version update(e.g. `openSUSE-Leap-15-4`). We don't want to deprecate major version directly.